### DISCUSSION TITLE
Redelegate peacefully....

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#91] [General]
![Screenshot 2023-05-25 at 22 08 03](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/416e3b91-022e-4679-8614-fc2d165d9648)

If you've reached this prompt, it means you've already made the decision to redelegate and have a specific validator in mind. Would it be possible to organize the validators alphabetically, so users don't have to scroll through the entire list to find their chosen validator? It would be much more convenient, especially if the new validator's name starts with a certain letter. Often, when going through the list, we tend to overlook the desired choice and have to start over from the beginning.

Alternatively, implementing a small input field where users can enter a maximum of three letters could be a great solution. For example, entering "P" would display only the validators whose names begin with the letter "P." This would greatly enhance the user experience!

![Screenshot 2023-05-25 at 22 07 49](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/6273112a-7d7a-4302-aaeb-d731a8a9c52b)

**Comment by TJ-BitCanna** [comment]
Good ideas. We'll explore the possibilities for a first release.

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- implement input field for people to search for a specific validator by typing the name
- think about either randomizing the list for redelegation or sorting alphabetically

**Comment by TJ-BitCanna** [comment]
Tested and approved.
